api/webpack-node.config.js

```diff
@@ -1,52 +1,65 @@
-/* eslint-disable */
 const path = require('path');
 const webpack = require('webpack');
 const nodeExternals = require('webpack-node-externals');
 const TerserPlugin = require('terser-webpack-plugin');
+const ForkTsCheckerWebpackPlugin = require('fork-ts-checker-webpack-plugin');
 
-const {
-  NODE_ENV = 'production'
-} = process.env;
+const { NODE_ENV = 'production' } = process.env;
 
 module.exports = {
   optimization: {
     minimize: true,
     minimizer: [
       new TerserPlugin({
+        parallel: true,
         terserOptions: {
-          sourceMap: true
-        }
-      })
-    ]
+          sourceMap: true,
+          compress: {
+            drop_console: true,
```

Comment:
I'm not 100% sure about whether this is a good idea but it does provide the biggest speedup of all these optimisations - @andywilkinshmcts what do you reckon?